roles/splunk_common/tasks/install_apps.yml

```diff
@@ -44,6 +44,8 @@
 
 - name: Check app contents
   shell: "set -o pipefail && tar --exclude='*/*/*' --exclude='*.*' -tf /tmp/app.spl | awk -F'/' '{ print$1 }' | uniq"
+  args:
+    executable: /bin/bash
```

Comment:
We might have some complaints about this by people who want to use other shells.

Reply:
Possibly, I would assume bash is available on our Docker images (because we install it) and most AWS AMIs. I'm not sure where else these Ansible plays are being used?

Otherwise, we could void the `set -o pipefail` and find ways to disable the linting.